testsuite/MDAnalysisTests/analysis/test_gnm.py

```diff
@@ -80,6 +81,18 @@ def test_generate_kirchoff(universe):
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
 
 
+def test_gnm_SVD_fail(universe):
+    gnm = mda.analysis.gnm.GNMAnalysis(universe)
+    # Initilize attributes like self._ts 
```

Comment:
Forgot to add this in my review, I guess auto-pep8 is complaining about it
```suggestion
    # Initilize attributes like self._ts
```